components/quest-boost/TokenSymbol.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent } from "react";
+import { CDNImage } from "@components/cdn/image";
+import { TOKEN_ADDRESS_MAP } from "@utils/constants";
+
+type TokenSymbolProps = {
+  tokenAddress: string;
+  imageDimensions?: { width: number; height: number };
+};
+const TokenSymbol: FunctionComponent<TokenSymbolProps> = ({
+  tokenAddress,
+  imageDimensions = { width: 20, height: 20 },
+}) => {
+  let tokenImageLink = "";
+  const network = process.env.NEXT_PUBLIC_IS_TESTNET ? "TESTNET" : "MAINNET";
```

Comment:
Is it possible to make this network const a global const (we're repeating this code all the time).

Reply:
done